lib/PersistedPaneset/PersistedPaneset.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useLocalStorage, writeStorage } from '@rehooks/local-storage';
+
+import { Paneset } from '@folio/stripes-components';
+
+const propTypes = {
+  appId: PropTypes.string.isRequired,
+  children: PropTypes.node,
+  id: PropTypes.string.isRequired,
+};
+
+const PersistedPaneset = ({
+  appId,
+  children,
+  id: panesetId,
+}) => {
+  const persistKey = `${appId}/${panesetId}/layout`;
```

Comment:
John brought up a good point that Jakub made earlier: we'll want to be sure to namespace localstorage by user also to account for the common use case that multiple users are using the same service point.